using-mobile-extensions/adobe-campaign-standard/README.md

```diff
@@ -18,43 +15,35 @@ This extension is considered beta functionality and is available only in Launch'
 
 ### Configure Campaign Standard Extension
 
-![](../.gitbook/assets/campaign-extension-config-v2.png)
+![](../../.gitbook/assets/campaign-extension-config-v4.png)
 
 #### ACS Endpoints
 
-{% hint style="info" %}
-Trying to find your ACS endpoint URLs? Contact your beta manager.
-{% endhint %}
-
-Type the endpoint URL\(s\) for your Adobe Campaign Standard instances. You can specify up to three unique endpoints for your development, staging, and production environments.
+Provide endpoint URL\(s\) for your Adobe Campaign Standard instances. You may specify up to three unique endpoints for your development, staging, and production environments.
 
 {% hint style="warning" %}
-For this extension, these ACS endpoint URLs should be typed in **without** the `http://` or `https://` and **cannot** end with a forward slash.
+
+For this extension, these endpoint URLs should be typed in **without** the `http://` or `https://.`and **cannot** end with a forward slash.
+
 {% endhint %}
 
 #### pKey
 
-A unique, auto-generated identifier for a mobile app configured in Adobe Campaign Standard. After you configured this extension in Launch, configure your Launch mobile property in Adobe Campaign Standard. When the configuration in Campaign is successful, pKeys are automatically generated for each Campaign Standard instance and configured in the Campaign extension for a successful validation.
+A unique, auto-generated identifier for a mobile app configured in Adobe Campaign Standard. After you've configured this extension in Launch, you will need to configure your Launch mobile property in Adobe Campaign Standard. When the configuration in Campaign is successful, pKey will be automatically generated, per Campaign Standard instance, and configured in Launch Campaign extension for successful validation.
```

Comment:
Please make the following changes:

- "you've" to "you configure".
- "you will need to" to "configure"